Hugo: template for shortcode "alert" not found

Hi,

I was trying to run the website locally.
But after I run `make`, Hugo returned an error.
It seems like theme `doks` is not downloaded?

```
make
cd themes/doks/ && \
	npm install && \
	rm -rf content

up to date, audited 1 package in 303ms

found 0 vulnerabilities
bash synchronize.sh
Cloning into 'repos/prometheus-operator'...
remote: Enumerating objects: 440, done.
remote: Counting objects: 100% (440/440), done.
remote: Compressing objects: 100% (395/395), done.
remote: Total 440 (delta 93), reused 139 (delta 21), pack-reused 0
Receiving objects: 100% (440/440), 3.04 MiB | 1.07 MiB/s, done.
Resolving deltas: 100% (93/93), done.
Cloning into 'repos/kube-prometheus'...
remote: Enumerating objects: 265, done.
remote: Counting objects: 100% (265/265), done.
remote: Compressing objects: 100% (240/240), done.
remote: Total 265 (delta 85), reused 75 (delta 17), pack-reused 0
Receiving objects: 100% (265/265), 272.33 KiB | 645.00 KiB/s, done.
Resolving deltas: 100% (85/85), done.
cp: images/custom-metrics-elements.png: No such file or directory
hugo
Start building sites …
Total in 60 ms
Error: Error building site: "/Users/leewei/dev/website/content/docs/prologue/introduction.md:31:1": failed to extract shortcode: template for shortcode "alert" not found
make: *** [public] Error 255
```

--- a/.gitignore
+++ b/.gitignore
@@ -6,6 +6,7 @@
 # Generated content to ignore (sorted alphabetically)
 
 /content/adopters.md
+/content/custom-metrics-elements.png
 /content/docs/developing-prometheus-rules-and-grafana-dashboards.md
 /content/docs/exposing-prometheus-alertmanager-grafana-ingress.md
 /content/docs/kube-prometheus-on-kubeadm.md
@@ -23,4 +24,3 @@
 /content/docs/operator/thanos.md
 /content/docs/operator/troubleshooting.md
 /content/docs/prologue/contributing.md
-/images/custom-metrics-elements.png

--- a/README.md
+++ b/README.md
@@ -3,7 +3,14 @@ The Prometheus Operator &amp; kube-prometheus website
 
 ## Development
 
-To generate the website you need to install [hugo](https://gohugo.io/).
+To generate the website you need to install the extended version of [Hugo](https://gohugo.io/).
+
+Before building the docs, you need to pull the [doks](https://getdoks.org/) theme locally:
+
+```shell
+git submodule init
+git submodule update
+```
 
 Fetching the theme, synchronizing the docs from both repositories and building it locally is done by running:
 
@@ -11,4 +18,4 @@ Fetching the theme, synchronizing the docs from both repositories and building i
 make
 ```
 
-If you want to run a local devleopment server you can simply run `hugo serve` after running `make` once.
+If you want to run a local development server you can simply run `hugo serve` after running `make` once.

--- a/synchronize.sh
+++ b/synchronize.sh
@@ -1,12 +1,14 @@
 #!/usr/bin/env bash
 
+set -xe
+
 rm -rf repos/
 mkdir repos/
 
-git clone git://github.com/metalmatze/prometheus-operator -b website --depth 1 repos/prometheus-operator
-git clone git://github.com/metalmatze/kube-prometheus -b website --depth 1 repos/kube-prometheus
+git clone git://github.com/prometheus-operator/prometheus-operator -b master --depth 1 repos/prometheus-operator
+git clone git://github.com/prometheus-operator/kube-prometheus -b main --depth 1 repos/kube-prometheus
 
-# promtheus-operator
+# prometheus-operator
 
 cp repos/prometheus-operator/ADOPTERS.md content/adopters.md
 cp repos/prometheus-operator/CONTRIBUTING.md content/docs/prologue/contributing.md
@@ -19,7 +21,7 @@ cp repos/prometheus-operator/Documentation/rbac.md content/docs/operator/rbac.md
 cp repos/prometheus-operator/Documentation/thanos.md content/docs/operator/thanos.md
 cp repos/prometheus-operator/Documentation/troubleshooting.md content/docs/operator/troubleshooting.md
 
-cp repos/prometheus-operator/Documentation/custom-metrics-elements.png images/custom-metrics-elements.png
+cp repos/prometheus-operator/Documentation/custom-metrics-elements.png content/docs/operator/custom-metrics-elements.png
 
 # kube-prometheus
 

